app/uk/gov/hmrc/apiplatformmicroservice/apidefinition/controllers/ApiDefinitionController.scala

```diff
@@ -37,15 +37,15 @@ class ApiDefinitionController @Inject() (
   )(implicit ec: ExecutionContext)
     extends BackendController(controllerComponents)
     with ActionBuilders
-    with FilterApis {
+ {
 
   import ApiDefinitionController.JsonFormatters._
   import ApiDefinitionController._
 
-  def fetchAllSubscribeableApis(applicationId: ApplicationId): Action[AnyContent] =
+  def fetchAllSubscribeableApis(applicationId: ApplicationId, unrestricted: Option[Boolean] = Some(false)): Action[AnyContent] =
```

Comment:
I got confused if this should default to true or false (I think it's correct to be false).

 - Would this be clearer if the name was 'restricted' to avoid the double negative?
`restricted: Option[Boolean] = Some(true)